api/test/e2e/label_test.go

```diff
@@ -302,6 +333,14 @@ func TestLabel(t *testing.T) {
 			Headers:      map[string]string{"Authorization": token},
 			ExpectStatus: http.StatusOK,
 		},
+		{
```

Comment:
Can `plugin_config` be updated? If so, we should also add update test suites.

Reply:
updated.